jdaviz/configs/default/plugins/subset_plugin/tests/test_subset_plugin.py

```diff
@@ -1,17 +1,18 @@
 import pytest
 from glue.core.roi import XRangeROI
+from jdaviz.configs.default.plugins import SubsetPlugin
 
 
 @pytest.mark.filterwarnings('ignore')
 def test_plugin(specviz_helper, spectrum1d):
     specviz_helper.load_spectrum(spectrum1d)
-    p = specviz_helper.plugins['Subset Tools']
+    p = specviz_helper.app._subset_tools_plugin
```

Comment:
is this just to force coverage of the property itself?  Maybe just add a note stating that's why (users should still only access it through `.plugins`, probably)

Reply:
what was the error?  I'm worried that could be a sign of something else wrong

Reply:
I tested and they both work, maybe I spelled something wrong the first time but it seems to work now.